.github/workflows/configlet-sync.yml

```diff
@@ -0,0 +1,87 @@
+name: Configlet Auto Sync
+
+on:
+  workflow_call:
+
+jobs:
+  sync-docs-metadata:
+    timeout-minutes: 10
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+
+      - name: Fetch configlet
+        run: ./bin/fetch-configlet
+
+      - name: Run configlet sync for files
+        run: ./bin/configlet sync --docs --metadata --filepaths -u -y
+
+      - name: Create pull request if changes
+        uses: peter-evans/create-pull-request@271a8d0340265f705b14b6d32b9829c1cb33d45e
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: "🤖 Auto-sync docs, metadata, and filepaths"
+          title: "🤖 Configlet sync: docs, metadata, and filepaths"
+          body: |
+            This PR was generated automatically by a scheduled workflow.
+
+            It includes updates from `configlet sync` for:
+            - 📄 Documentation
+            - 🧭 Metadata
+            - 🗂️ Filepaths
+
+            Please review and merge if everything looks good!
+          branch: configlet-auto-sync
+          delete-branch: true
+
+  check-test-sync:
+    timeout-minutes: 10
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+
+      - name: Fetch configlet
+        run: ./bin/fetch-configlet
+
+      - name: Run configlet sync for test and capture output
+        id: sync_test
+        shell: bash {0}
+        run: |
+          configlet_raw_output="$(./bin/configlet sync --tests 2>&1)"
+          exit_code=$?
+          echo "exit_code=$exit_code" >> "$GITHUB_OUTPUT"
```

Comment:
If you want to be consistent, you can replace all the `echo` lines with `printf` ... though it doesn't especially matter either way.